http/server.go

```diff
@@ -112,3 +113,17 @@ func GetRequestID(r *http.Request) string {
 func GetUserID(r *http.Request) string {
 	return strx.Or(r.Header.Get("X-User"), r.Header.Get("X-User-Id"))
 }
+
+// ReadSkipAndCount returns the skip and count pagination values from the query
+func GetSkipAndCount(r *http.Request) (skip int, count int, exists bool) {
+	skip, err := strconv.Atoi(r.URL.Query().Get("skip"))
+	if err != nil {
+		skip = 0
+	}
+	count, err = strconv.Atoi(r.URL.Query().Get("count"))
```

Comment:
Can we introduce a hard limit? 200 was what we agreed to in slack. `math.Min(count, 200)`

Reply:
Done